cli/cmd/cluster_create.go

```diff
@@ -68,7 +71,13 @@ func (r *runners) createCluster(_ *cobra.Command, args []string) error {
 	}
 	cl, err := r.createAndWaitForCluster(opts)
 	if err != nil {
-		return err
+		if errors.Cause(err) == ErrWaitDurationExceeded {
+			defer func() {
+				os.Exit(124)
```

Comment:
wont this print "Dry run succeeded." since you defer here?

Reply:
In theory, yes. But combining DryRun with Wait doesn't make a lot of sense. It actually should be impossible probably.